kola-denylist.yaml

```diff
@@ -0,0 +1,2 @@
+- pattern: ext.config.version.rhaos-pkgs-match-openshift
+  tracker: https://issues.redhat.com/projects/OCPBUGS/issues/OCPBUGS-58271
```

Comment:
That bug looks unrelated.

Reply:
Yeah, we pinged ART about this issues and put a placeholder bug in the meantime. I guess a comment would have been useful

Reply:
It could also just be a ticket against this repo.